sdk/include/xdk/ntdef.template.h

```diff
@@ -220,7 +220,7 @@ char _RTL_CONSTANT_STRING_type_check(const void *s);
 #endif
 #define RTL_CONSTANT_STRING(s) { \
     sizeof(s)-sizeof((s)[0]), \
-    sizeof(s) / sizeof(_RTL_CONSTANT_STRING_type_check(s)), \
+    sizeof(s) / (sizeof(_RTL_CONSTANT_STRING_type_check(s))), \
```

Comment:
What kind of error does Clang emit here? (these extra parentheses look really superfluous)

Reply:
See the commit message?
It includes the full output

Reply:
Hm yeah; the construct is actually quite strange. Was it based on what MS' WDK does have?

Reply:
Yes, it's based on WDK headers.
It's simply a `/ 1`, i.e. has no effect -- but the `_RTL_CONSTANT_STRING_type_check` makes it type-safe in C++.